bls/src/test/java/tech/pegasys/teku/bls/BLSPublicKeyTest.java

```diff
@@ -29,16 +31,18 @@
           "0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000");
 
   @Test
-  void isValidReturnsTrueForValidKey() {
-    BLSPublicKey publicKey = BLSPublicKey.random(1);
-    assertTrue(publicKey.isValid());
+  void fromBytesCompressedValidate_okWhenValidBytes() {
+    BLSPublicKey.fromBytesCompressedValidate(BLSPublicKey.random(1).toBytesCompressed());
```

Comment:
Should we have an `assert` here, if only to be explicit about what we are testing for?